iib/common/tracing.py

```diff
@@ -0,0 +1,163 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+"""Configures the Global Tracer Provider and exports the traces to the OpenTelemetry Collector.
+
+The OpenTelemetry Collector is configured to receive traces via OTLP over HTTP.
+The OTLP exporter is configured to use the environment variables defined in the ansible playbook.
```

Comment:
I think this line still needs to be changed. We should simply have `The OTLP exporter is configured to use environment variables explained in README.md`

Reply:
@yashvardhannanavati - can you schedule a call to go over all the changes. It's a bit of back and forth to be making the changes everytime.